sippy-ng/src/releases/TopLevelIndicators.js

```diff
@@ -59,22 +81,98 @@ export default function TopLevelIndicators(props) {
         </Typography>
       </Grid>
 
-      <Grid item md={3} sm={6}>
-        <SummaryCard
-          key="infrastructure-summary"
-          threshold={INFRASTRUCTURE_THRESHOLDS}
-          name="Infrastructure"
-          link={
-            '/tests/' +
-            props.release +
-            '/details?test=[sig-sippy] infrastructure should work'
-          }
-          success={props.indicators.infrastructure.current.percentage}
-          fail={100 - props.indicators.infrastructure.current.percentage}
-          caption={indicatorCaption(props.indicators.infrastructure)}
-          tooltip="How often we get to the point of running the installer. This is judged by whether a kube-apiserver is available, it's not perfect, but it's very close."
-        />
-      </Grid>
+      {newInstall ? (
+        <Grid item md={3} sm={6}>
+          <SummaryCard
+            key="infrastructure-summary"
+            threshold={INFRASTRUCTURE_THRESHOLDS}
+            name="Infrastructure"
+            link={
+              '/tests/' +
+              props.release +
+              '/details?test=cluster install.install should succeed: infrastructure'
```

Comment:
These links don't work since the test details page was changed to show the NURP's.

I'd suggest linking to them like this: https://github.com/xueqzhan/sippy/commit/d949d851c92192bb04fd496305381c1d47f132cb